README.rst

```diff
@@ -1,5 +1,13 @@
-p5
-===
+.. raw:: html
```

Comment:
I think it's better to keep using markdown whenever possible because it offers better readability. This would mean we'll lose centering, but I think it's fine and matches with the general aesthetics of Github.

Reply:
Great 👍🏻

Reply:
Oops meant to say reStructuredText instead of markdown